src/compiler/parser/index.js

```diff
@@ -33,7 +33,7 @@ const dynamicArgRE = /^\[.*\]$/
 const argRE = /:(.*)$/
 export const bindRE = /^:|^\.|^v-bind:/
 const propBindRE = /^\./
-const modifierRE = /\.[^.]+/g
+const modifierRE = /\.[^.\]]+(?=\.|$)/g
```

Comment:
`.a` in `'v-my:[deep.a.b]'` will be matched as a modifier.

Reply:
Sorry, this is my mistake, I updated the regular expression again. And modified the `parseModifiers` method for new regular expressions. If there is any problem, please let me know.